File: Java/Practice/p2.java
package Practice;


import java.util.Scanner;

import static java.lang.Math.*;


public class p2 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Введите a: ");
        double a = in.nextDouble();
        System.out.print("Введите b: ");
        double b = in.nextDouble();
        System.out.print("Введите c: ");
        double c = in.nextDouble();
        System.out.print("Введите x: ");
        double x = in.nextDouble();
        double y;
        if (x == a && x == b && x == c) {
            y = sin(x);
        } else if (x <= a && b <= x && x <= c) {
            y = cos(x);
        } else {
            y = abs(x);
        }
        System.out.println("y = " + y);
    }
}

File: Python/p2.py
import math
a = float(input("a = "))
b = float(input("b = "))
c = float(input("c = "))
x = float(input("x = "))
if x == a and x == b and x == c:
    y = math.sin(x)
elif x <= a and b <= x <= c:
    y = math.cos(x)
else:
    y = abs(x)
print("y =", y)
